business_requirement_task_type/__openerp__.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# © 2015
```

Comment:
one line for the company copyright:
© 2015 Elico Corp, contributor: Eric Caudal, Alex Duan, Xie XiaoPeng